src/util.py

```diff
@@ -260,12 +265,145 @@ def xr_add_cyclic_longitudes(da, coord):
 
     return new_da
 
+def findLocalExtrema(da, maxVal=1040, minVal=975, eType='Min'):
```

Comment:
This is an arbitrary change, but consider changing any use of `max` and `min` in variable names to `high` and `low`. Since our typical user will be in atmospheric or earth sciences, those terms may make more sense given that this function is designed to find highs and lows.